mk/flags.mk

```diff
@@ -6,8 +6,7 @@ OBJCOPY = $(CROSS_COMPILE)objcopy
 GDB = $(CROSS_COMPILE)gdb
 HOSTCC  = gcc
 WGET = wget
-# FIXME: check version >= 3.5
-PYTHON = python3
+PYTHON ?= python
```

Comment:
I remember when I use `python3` is because on ubuntu and macOS the default `python` is `python2`, not 3. If we make this change, should we also add some documentation or notice for users on these platforms? 

e.g. `Error: Python version must >= 3.5, use PYTHON=/python/binary/path`

Reply:
Yes, but `python3` do not guarantee its version >= 3.5,
if we use `python3`, Travis will complain the version is below 3.5.
Also, not all the system have `python3`, it's only a symbolic link. I think `python` is more general than `python3`.
If `python3` is better, why the `readme.md` shows `python -m tests` and `python -m tests fs_1 cond_2`?


`Error: Python version must >= 3.5, use PYTHON=/python/binary/path`,
this error message remind users of their python version.

We can add a topic like `Environment` to explicit instruct the tool or library version PikoRT needed.

Reply:
I'm not saying `python3` is better than `python`, but we still need to think a way to solve this situation.

`README.md` using `python` because my system using `python` as python3, but on Travis that time, it will need to use `python3` to explicit to use python3.

Can we make some trick like [this](https://stackoverflow.com/a/22105036), and maybe fallback to `python3` when `python` is not 3?

Reply:
I think the `Environment` will need to be added if we rely on env parameters.

Reply:
> README.md using python because my system using python as python3, but on Travis that time, it will need to use python3 to explicit to use python3.

umm.. `python3` in Travis is not >=3.5. Assigning `python: 3.6` in `.travis.yml` changes `python` version from 2.X.X to 3.6.X.
What you want needs to explicit `apt-get python3.6` in `.travis.yml` and make sure `python3` is symbolic link to `python3.6`, so that I can safely rollback to `python3` and Travis is happy.

>Can we make some trick like this, and maybe fallback to python3 when python is not 3?

Even I make `Makefile` automatically changes `$PYTHON` can't promise that its version is correct but adding code complexity of `Makefile`.
I think keep `python` is better.


>I think the Environment will need to be added if we rely on env parameters.

In `Build Piko/RT - Prerequisites` is where you should mention the environment, not only said we need to apt-get this tool but explicitly tell the version of the tool.
Maybe a year later, the version of tools which `apt-get`ed  is not suitable for PikoRT